### PR TITLE
Update planner comments with cautionary info

### PR DESCRIPTION
**Pull request by Asura5**
## Link to JIRA ticket

Add a link to the JIRA ticket that the changes in this PR describe.

### *What is the problem we're trying to solve*

Add a summary of the problem and any additional context that helps explain what the issue we're trying to solve is.

### *What changes have we introduced*

Add descriptions of what changes were made to address the problem and provide a solution.

Include any links to commits that introduce significant additions of code if they help explain the process of coming to the solution. e.g Addition of test database setup, addition of first test and production class, removal of buggy code, etc.

#### _Checklist_

- [x] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [x] Added tests to cover all new production code
- [x] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [x] Checked all code for possible refactoring
- [x] Code pipeline builds correctly

### *Follow up actions after merging PR*

Are there any next steps that need to addressed after merging this PR? Add them here.

### CODE REVIEW

File: RepairsApi/V2/Notifications/DRSNotificationHandler.cs [hunk 1]
```diff
@@ -61,5 +61,14 @@ public async Task Notify(WorkOrderCompleted data)
             }
             await DrsService.CompleteOrder(data.WorkOrder);
         }
+
+        public async Task Notify(WorkOrderUpdated data)
+        {
+            if (await UseDrs(data.WorkOrder))
```
Comment (LBHBomotoso) [inline RepairsApi/V2/Notifications/DRSNotificationHandler.cs]: Is it the static analysis that stops the negated condition being checked?

File: RepairsApi/V2/Generated/DRS/DrsSoapPartials.cs [hunk 2]
```diff
@@ -22,9 +24,9 @@ public partial class SOAPClient
     }
 
     // ReSharper disable once InconsistentNaming
-    public partial class BookingCode : entity
+    public partial class bookingCode : entity
     {
-        public BookingCode()
+        public bookingCode()
```
Comment (LBHBomotoso) [inline RepairsApi/V2/Generated/DRS/DrsSoapPartials.cs]: Why are the names changed?

Comment (Asura5) [reply RepairsApi/V2/Generated/DRS/DrsSoapPartials.cs]: Reverted, overzealous IDE

File: RepairsApi/V2/Notifications/DRSNotificationHandler.cs (hunk 1 above)
Comment (Asura5) [reply RepairsApi/V2/Notifications/DRSNotificationHandler.cs]: This method has been removed

File: RepairsApi/V2/Services/DRS/DrsService.cs [hunk 3]
```diff
@@ -109,6 +116,15 @@ public async Task CompleteOrder(WorkOrder workOrder)
             _logger.LogInformation("DRS completed work order {WorkOrderId}", workOrder.Id);
         }
 
+        private void LogResponseOnFailure(updateBookingResponse response)
```
Comment (josh-dadswell-hackney) [inline RepairsApi/V2/Services/DRS/DrsService.cs:119]: Dont see this function being usedc anywhere, am I blind?

**Review by josh-dadswell-hackney — APPROVED** [review]

File: RepairsApi/V2/Services/DRS/DrsService.cs (hunk 3 above)
Comment (Asura5) [reply RepairsApi/V2/Services/DRS/DrsService.cs]: Removed

**Pull request merged**